### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by Johnadibe**
- Added calculate.js and operate.js files into logic directory.
- Implemented click event handlers using the math logic from calculate.js and operate.js files.
- Tested the  Calculator app in the browser to ensure it can perform math operations.
- Used class based component to implement the functionality.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,33 +1,60 @@
...
+
+    render() {
+      const { total, next, operation } = this.state;
+      return (
+        <div className="container">
+          <div className="output" onChange={this.resultDisplay}>
+            {total}
+            {operation}
+            {next}
+          </div>
```
Comment (AmaduKamara) [inline src/components/Calculator.js]: - [ ] For this milestone, you're required to handle state management with the `useState` hook.
Below is a sample of how your component should look like:
```javascript
// ./src/components/Calculator.js

import React, { useState } from 'react';
import calculate from ../logic; // Import math logic

function Calculator() {
    const [total, setTotal] = useState(0);
    return {
      <div>
       // Calculator app 
      </div>
    };
}
```
**Your Project requirement**

Screenshot 2023-02-16 at 2 04 15 PM

Screenshot 2023-02-16 at 2 06 25 PM

- [ ] Kindly make use of the react useState hook 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @Johnadibe,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used math logic files provided by Microverse 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 1 above)
Comment (Johnadibe) [reply src/components/Calculator.js]: Hello @AmaduKamara, 

Thank you for your review but Please Sir using useState Hook is the fourth project of the week which is Hooks project refactoring from class based component to functional component and not for the third day project. That was actually the project requirement.

Comment (rloterh) [reply src/components/Calculator.js]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi John,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Event handling properly implemented!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**